backend/src/api/auth/auth.routers.js

```diff
@@ -0,0 +1,41 @@
+const { Router } = require("express");
+const authRouter = Router();
+const Joi = require("joi");
+const { validate } = require("../../utils.js/validate");
+
+const AuthController = require("./auth.controller");
+const { authorize } = require("../../utils.js/authMiddleware");
+const catchAsync = require('../../utils.js/catchAsync');
+
+
+
+const registerScheme = Joi.object({
```

Comment:
`Schema`. Please fix in other places also

Reply:
ok